examples/python/szvav_sup.py

```diff
@@ -89,28 +93,34 @@ def run(plot=False, customized_kpi_config=None):
     start = time.time()
     # Initialize test case
     print('Initializing test case simulation.')
-    res = requests.put('{0}/initialize'.format(url), data={'start_time':0,'warmup_period':0}).json()
-    if res:
+    res = requests.put('{0}/initialize'.format(url), json={'start_time':0,'warmup_period':0}).json()
```

Comment:
Since ``json`` is imported, I think it is better to keep with ``data``.  Any reason you suggest the change?

Reply:
sorry for the confusion. I used ``json`` as it is necessary for our eplus docker server. I will change it back.